requirements.txt

```diff
@@ -2,5 +2,5 @@ Click<7
 coloredlogs
 marshmallow
 pyyaml
-SQLAlchemy<=1.4
+SQLAlchemy>=1.3.0,<1.5.0
```

Comment:
```suggestion
SQLAlchemy>=1.4.0,<1.5.0
```

Reply:
We need to keep allowing for 1.3 since the version in housekeeper is locked to the one in cg